packages/client/src/generation/generateClient.ts

```diff
@@ -121,9 +119,10 @@ export async function buildClient({
   fileMap['package.json'] = JSON.stringify(
     {
       name: GENERATED_PACKAGE_NAME,
-      main: 'index.js',
-      types: 'index.d.ts',
-      browser: 'index-browser.js',
+      types: clientPackage.types,
+      main: clientPackage.main,
+      browser: clientPackage.browser,
+      exports: clientPackage.exports,
```

Comment:
Parts of the `package.json` of `@prisma/client` is re-used by the generated client `package.json`, useful for the map.